libnet/configure.ac

```diff
@@ -4,25 +4,50 @@ dnl All rights reserved.
 dnl
 dnl Process this file with autoconf to produce a configure script.
 
-AC_INIT([libnet],[1.2-rc3])
+AC_INIT([libnet],[1.2-rc3],,,[http://github.com/sam-github/libnet])
 AC_MSG_RESULT(beginning autoconfiguration process for libnet-${PACKAGE_VERSION} ...)
 AC_CANONICAL_TARGET
 AC_CONFIG_SRCDIR([src/libnet_build_ip.c])
 AC_CONFIG_MACRO_DIR([m4])
-AM_INIT_AUTOMAKE
+AM_INIT_AUTOMAKE([foreign subdir-objects no-dependencies -Wall])
```

Comment:
I can't really push my changes yet because they're a little messy right now, but here I have:
`AM_INIT_AUTOMAKE([1.14 foreign no-dependencies nostdinc tar-ustar dist-zip])`

`nostdinc` is to reduce the number of unnecessary and redundant include flags during build